### DISCUSSION TITLE
Lenovo T580 / i5 8350u / chipset GPU 620

### DISCUSSION THREAD

**Discussion by MalikBehloul** [MultimediaLucario/Lenovo-ThinkPad-T480#6] [Show and tell]
Thank you very much  this EFI is very good for my Lenovo T580 !!! ;)

**Comment by MultimediaLucario** [comment]
No problem.

**Reply by MalikBehloul** [reply]
Hello the new version Efi Sonoma works better the bluetooth is more stable we gain speed on the T580 thank you very much for your work ;)

**Reply by MalikBehloul** [reply]
loss of wifi and apple mouse when returning from sleep

**Comment by MalikBehloul** [comment]
hello I made changes in the ACPI you can try these files thank you
[dossier.zip](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T480/files/15073568/dossier.zip)
